src/Symfony/Component/Validator/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 -----
 
  * added options `iban` and `ibanPropertyPath` to Bic constraint
+ * added support for Vatican City State IBAN format
```

Comment:
Should be removed when rebased on 3.4 (as this should indeed be considered as a bug fix).